bin/tachyon-mount.sh

```diff
@@ -213,6 +213,7 @@ case "${1}" in
         mount_local $1
         ;;
       workers)
+        bin=`cd "$( dirname "$0" )"; pwd`
         $LAUNCHER $bin/tachyon-workers.sh $bin/tachyon-mount.sh $1
```

Comment:
Sure @uronce-cc , the different thing on `init_env` it is that it have some variable definitions and a sanity check on memory worker which this is good for local worker but with remote workers, they will inherit $TACHYON_WORKER_MEMORY_SIZE from their local `libexec/tachyon-config.sh` . Was my understanding right?